code/modules/mob/mob.dm

```diff
@@ -220,7 +220,7 @@
 			if(M != loc) // Only give the blind message to hearers that aren't the location
 				msg = blind_message
 				msg_type = MSG_AUDIBLE
-		else if(M.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,M)) //if it is too dark, unless we're right next to them.
+		else if(M.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,M) && !isobserver(M)) //if it is too dark, unless we're right next to them.
```

Comment:
This should probably be some trait/flag so it's more obvious why this check is here. Or add a comment at least.

Reply:
Good point